### PR TITLE
[VEGA-2839] fix(components): Исправление очистки формы при закрытии модального окна

### PR DESCRIPTION
**Pull request by lipgen**
### Description
- Поправил сброс формы при закрытии по нажатию вне модального окна и при добавлении группы объектов
- Добавил trim на onBlur для текстового поля, сделал опциональным по передаче ```trimOnBlur: boolean``` 

### Checks
- [x] Jira link attached: [VEGA-2839](https://jira.vegaspace.tk/browse/VEGA-2839)
- [ ] Possible Associated Jira links:
- [x] Personal Vega Instance link attached: http://ltumoyakov-vega2839-lanit-vega-builder.code013.org
- [ ] Jira tasks for technical debt were created (if needed)

### Type
- [ ] Feature
- [ ] Alpha Feature ([Alpha] MR/PR Title)
- [x] Bug Fix
- [ ] Test
- [ ] Refactoring

### PR/MR Requirements
- [x] PR/MR title meet requirements/convention
- [x] Target branch is correct
- [x] PR/MR branch was rebased at correct branch
- [x] Diff was checked
- [ ] Configuration files were checked at test repositories
- [x] [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)

### General
- [ ] Exploratory Tested at Personal Vega Instance
- [x] Existing E2E Tests done, Screenshot attached
- [ ] Existing Integration Tests done, Screenshot attached, Jenkins link:
- [x] Existing Unit Tests done, Screenshot attached
- [ ] New E2E Tests developed
- [ ] New Integration Tests developed
- [x] New Unit Tests developed
- [ ] API documentation was fully and correctly updated

### E2E Suite Jenkins link
http://jenkins-remote-ci-gpn-vega-builder.code013.org:8081/job/STAND%20TOOLBOX/job/scenario-tests-on-demand/291/

### E2E Suite Screenshot
изображение

### Unit Tests Screenshot
изображение

### CODE REVIEW

File: src/components/objects-group/ObjectsGroupDialog.tsx [hunk 1]
```diff
@@ -45,14 +46,17 @@ export const ObjectsGroupDialog: React.FC = () => {
     ],
   });
 
-  const handleCloseDialog = (): void => {
+  const handleCloseDialog = (form: FormApi<NewGroupParams, Partial<NewGroupParams>>): void => {
+    setTimeout(form.reset);
```
Comment (DmitriiBo) [inline src/components/objects-group/ObjectsGroupDialog.tsx:50]:
```suggestion
     dispatch(toggleDialog(false));
    form.reset()
```

Comment (DmitriiBo) [reply src/components/objects-group/ObjectsGroupDialog.tsx:50]: А так не будет работать?

**Review by DmitriiBo — APPROVED** [review]

File: src/components/objects-group/ObjectsGroupDialog.tsx (hunk 1 above)
Comment (lipgen) [reply src/components/objects-group/ObjectsGroupDialog.tsx:50]: нет, он выдает ошибку, что нельзя сбрасывать форму во время submit, и предлагает такое решение с таймаутом

Comment (DmitriiBo) [reply src/components/objects-group/ObjectsGroupDialog.tsx:50]: а, окей

**Review by ababakov — APPROVED** [review]

**Pull request merged**